lib/graphql.ex

```diff
@@ -49,6 +55,15 @@ defmodule BridgeEx.Graphql do
       else
         defp format_response({ret, response}), do: {ret, response}
       end
+
+      defp with_authorization_headers(headers, nil), do: {:ok, headers}
```

Comment:
Visto che sei in una macro non vale la pena di compilare direttamete la funzione come fa sopra?

Reply:
Grazie ❤️ 
Se ho capito bene il punto, dovrebbe essere risolto con 105a71fb819095a6a85dacd91b9ee336a8318776